lib/generators/alchemy/install/templates/menus.yml.tt

```diff
@@ -1,6 +1,7 @@
 # == In this configuration, you set up Alchemy's menu names.
 #
-# For further information please see http://guides.alchemy-cms.com/stable/menus.html
+# TODO: Ensure there is a working link with explanation on how menu's are constructed. Link below is broken.
+# For further information please see http://guides.alchemy-cms.com/menus.html
```

Comment:
There is no guide unfortunately. Can we remove the comment altogether for now?

Reply:
I am a new person testing out Alchemy having this undocumented is super confusing as this is very core to any webpage. 

After looking a bit on the code I found what to do.

What about instead having the links say: To create the views for the menu use `rails g alchemy:menus` and render the menu with <%= render_menu('menu_name') %>

At least this comment with get the people going for common use cases.